src/sentry/static/sentry/app/components/bases/pluginComponentBase.tsx

```diff
@@ -11,17 +11,28 @@ import {
 } from 'app/actionCreators/indicator';
 import {t} from 'app/locale';
 
-const callbackWithArgs = function (callback, ...args) {
+const callbackWithArgs = function (context: any, callback, ...args) {
   if (isFunction(callback)) {
-    callback = callback.bind(this, ...args);
+    callback = callback.bind(context, ...args);
```

Comment:
as you are touching this file maybe you could use ternary and remove this else? :)